tests/_support/AcceptanceTester.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Inherited Methods.
```

Comment:
Please elaborate on this further, as that will help us understand these methods.

Reply:
These methods were autogenerated when I bootstrapped the test suite.